content/sensu-core/1.6/faq.md

```diff
@@ -188,6 +188,22 @@ See the below table for a listing of services, ports, and protocols Sensu uses.
 | Sensu Client Socket               | TCP      | 3030 |
 | Uchiwa/Sensu Enterprise Dashboard | TCP      | 3000 |
 
+> What configuration files does Sensu require?
+
+See the table below for the location of the respective files needed:
+
+| Filename      | Client | Server |
+|---------------|--------|--------|
+| api.json      |        | ✅       |
+| client.json   | ✅     | ✅       |
+| config.json (see note)  |        |        |
+| dashboard.json|        | ✅       |
+| rabbitmq.json | ✅       |        |

```

Comment:
It seems odd that the server wouldn't require the rabbitmq config. Are we sure about this?

Reply:
Derp